src/site/stages/build/drupal/assets.js

```diff
@@ -46,18 +59,20 @@ function convertAssetPath(drupalInstance, url) {
   return `/files/${path}`;
 }
 
-function updateAttr(attr, doc, client) {
+function updateAttrAllEnvs(attr, doc) {
   const assetsToDownload = [];
-  doc(`[${attr}^="${client.getSiteUri()}/sites"]`).each((i, el) => {
-    const item = doc(el);
-    const srcAttr = item.attr(attr);
-    const newAssetPath = convertAssetPath(client.getSiteUri(), srcAttr);
-    assetsToDownload.push({
-      src: srcAttr,
-      dest: newAssetPath,
+  siteURIs.forEach(siteURI => {
+    doc(`[${attr}^="${siteURI}/sites"]`).each((i, el) => {
+      const item = doc(el);
+      const srcAttr = item.attr(attr);
+      const newAssetPath = convertAssetPath(siteURI, srcAttr);
+      assetsToDownload.push({
+        src: srcAttr,
+        dest: newAssetPath,
+      });
+
+      item.attr(attr, newAssetPath);
```

Comment:
I'm curious if you think there would be any benefit to stringify-ing and then doing the find-and-replace with the GraphQL response [JSON](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/drupal/api.js#L64). It seems like our handling of the CMS assets is pretty complex right now, not that it's a simple task to do anyway.

Reply:
I'm not sure I see any benefit to that over what we have, but I'm not necessarily against it. Though I think the way we have the client fetch and data manipulation separated now is pretty good.